ui.frontend/src/main/webpack/components/_customsearch.scss

```diff
@@ -0,0 +1,52 @@
+@import "https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css";
+
+.cmp-customsearch {
+      margin: 20px auto;
+      width: 100%;
+
+      &__container {
+            display: flex;
+            margin: 0px auto;
+            width: 70%;
+
+            &-input {
+                  border: none;
+                  font-family:'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+                  outline: none;
+                  padding: 10px 20px;
+                  width: 92%;
+            }
```

Comment:
Make sure to use media query to make your UI responsive

Reply:
Done with Media query.